pkg/stanza/fileconsumer/testdata/config.yaml

```
@@ -54,15 +54,17 @@ exclude_one:
 sort_by_numeric:
   type: mock
   ordering_criteria:
+    regex: 'err\.(?P<value>[a-zA-Z])\.\d+\.\d{10}\.log'
```

For clarity of the example, let's name the captured field in a way that reflects the meaning of the value.
```suggestion
    regex: 'err\.(?P<file_num>[a-zA-Z])\.\d+\.\d{10}\.log'
```

---

Updated.